test/test_jit.py

```diff
@@ -4189,25 +4189,48 @@ def __init__(self, b=None):
             def forward(self, input, opt=None):
```

Comment:
I think this might be easier to test if each condition was tested separately in different function, and you could try to remove the expect file at that point